finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,13 @@
+var tenSpeckledFrogs
+
+
+
+ for (var i = 10; i > 0; i--){
```

Comment:
i++ is more common to see used, but I understand why it can be helpful to use i-- as it avoids doing some math to print the correct number. Nice job seeing the special conditions you need to use! I like to put the default condition in the last `else` block- the one that will be hit most often, in this case the one you have in the `if` block.